Source/SDFutureExtensions/Private/FutureExtensionsStaticFuncs.cpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "FutureExtensions.h"

```

Comment:
This introduces a compile error based on the order of compilation - recompiling does solve it but it indicates a unity build issue.
It expects the corresponding header to be the first include, but the plugin is written to assume the include order defined in FutureExtensions.h.

Any thoughts on how to solve this.

Reply:
I've got changes to convert to a more traditional include order. I can add these if required.